lib/assessments/permission.rb

```diff
@@ -73,23 +77,23 @@ def revoke_level(user)
           assessment.viewers.destroy(user)
         when :network_partner
           assessment.network_partners.destroy(user)
-        assessment.reload
+          assessment.reload
```

Comment:
Do you mean that is looks like it was supposed to reload inside `update_level` either way?